doc/pyang.1.dbk

```diff
@@ -305,6 +305,30 @@
         </listitem>
       </varlistentry>
 
+      <varlistentry>
+        <term>
+          <option>--msg-template</option>
+          <replaceable>msg-template</replaceable>
+        </term>
+        <listitem>
+          <para>
+            Print out error message in defined <replaceable>msg-template</replaceable>.
+          </para>
+          <para>
+            Template used to display error messages. This is a python
+            new-style format string used to format the message
+            information with keys file, line, code, type, and msg.
+            Example:
```

Comment:
remove this line

Reply:
Done. 
* Removed line from man page
* move test cases to test_issues
* Renaming of directory to test_i557 to reference to pull request id

Reply:
You removed to much from the man page :)   I just meant that you should remove the line "Example:", since the next line also says the same thing.  So please put lines 318-320 above back.

Reply:
haha, misunderstood. Reverted the mentioned lines. Tests passed. Cheers